docs/faq.md

```diff
@@ -214,6 +214,7 @@ spec:
     - name: JAVA_TOOL_OPTIONS
       value: <JVM flags>
 ```
+Note that the `JAVA_TOOL_OPTIONS` environment varible only supports a max length of **1024** characters. Anything longer than this will be cut off by the JVM.
```

Comment:
```suggestion
Note that many JVMs may only support a max length of **1024** characters for the `JAVA_TOOL_OPTIONS` environment variable, and anything longer than this may be cut off by the JVM.

For Java 9+, often you may want to use [`JDK_JAVA_OPTIONS`](https://stackoverflow.com/questions/52986487/what-is-the-difference-between-jdk-java-options-and-java-tool-options-when-using) instead of `JAVA_TOOL_OPTIONS`.
```

Reply:
thanks for your suggestion. I actually checked newer JDK versions and those don't seem to have that limitation. Still I think it's worth mentioning this limit